src/app.js

```diff
@@ -462,6 +462,7 @@ var run = function () {
       $('#ghostbar').remove()
       $(document).unbind('mousemove')
       dragging = false
+      delta = (delta < 50) ? 50 : delta
```

Comment:
I have just checked the rhs bar does need about 315px. If smaller not all tabs are shown.